docs/cloudsync_mqtt.md

```diff
@@ -0,0 +1,272 @@
+<img src="images/CloudSync/MQTTArchitecture.png" alt="image" align="left"/>
+
+# 2. EC2 Instance Setup Description
+### A. Create a Role for IOT Access
+1. Go to the AWS Web Console and click create a Role
+<img src="images/CloudSync/EC2-1.png" alt="image" style="margin:10px"/> 
+2. Select EC2 and click on Next: Permissions 
+<img src="images/CloudSync/EC2-2.png" alt="image" style="margin:10px"/>
+3. Filter with the value AWSIoTConfigAccess. Then select the policy AWSIoTConfigAccess and click on Next: Tags. Skip the next screen by clicking on Next: Review.
+<img src="images/CloudSync/EC2-3.png" alt="image" style="margin:10px"/>
+4. Enter AWS_IoT_Config_Access as the Role name and enter a Role description. Review the role and click on Create role
+<img src="images/CloudSync/EC2-4.png" alt="image" style="margin:10px"/>
+5. Now that the Role has been created you can go to Amazon EC2.
+
+### B. Creating EC2 Instance with role created
+
+1. Choose a region, in this article I am using N. Virginia (us-east-1). Then click on Launch Instance and use the filter with the value “ubuntu”. Select the Ubuntu Server 18.04 LTS x86 
+<img src="images/CloudSync/EC2-5.png" alt="image" style="margin:10px"/>
+2. Select the t2.micro instance type 
+<img src="images/CloudSync/EC2-6.png" alt="image" style="margin:10px"/>
+3. Click on Next: Configure Instance Details. In the IAM Role dropdown, select AWS_IoT_Config_Access
+Make sure you use the default VPC and that the Auto-assign Public IP is Enable to get a public IP automatically. If you wish to use another VPC, make sure the subnet you choose will enable you to remotely connect to your Amazon EC2 instance. Then, click on Next: Add Storage.
+<img src="images/CloudSync/EC2-7.png" alt="image" style="margin:10px"/>
+4. Leave everything as is and click on Next: Tag Instance. You may assign a tag to your instance. Click on Next: Configure Security Groups. Create a new security group as described in the screenshot
+<img src="images/CloudSync/EC2-8.png" alt="image" style="margin:10px"/>
+5. Review and launch the EC2 instance. Make sure to select an existing Key Pair or to create a new one in order to connect to the Amazon EC2 instance later on. Once the Amazon EC2 instance is running, click on “Connect” and follow instructions to establish a connection through a terminal.
+<img src="images/CloudSync/EC2-9.png" alt="image" style="margin:10px"/>
+
+
+## 3. MQTT Broker Installation on AWS
```

Comment:
```suggestion
# 3. Installing MQTT Broker on AWS
```

Reply:
DOne!!!

Reply:
Please revise this line with only one `#`.

Reply:
Done!!